src/fides/ctl/core/config/__init__.py

```diff
@@ -108,6 +108,45 @@ def censor_config(config: FidesConfig) -> Dict[str, Any]:
     return filtered
 
 
+def build_config(config_dict: Dict[str, Any]) -> FidesConfig:
```

Comment:
@sanders41 I think I finally found the solution to our config woes...the tests pass at least